### ISSUE TITLE
5.3.0 causes oktaAuthError version warning with okta-react 6.1

### ISSUE THREAD

**Issue by msherman** [okta/okta-auth-js#897]
I am receiving the following error message when trying to run the okta authentication flow within react with version 5.3.0 of the okta-auth-js lib.

```
AuthSdkError: Passed in oktaAuth is not compatible with the SDK, okta-auth-js version 5.x is the current supported version.
```

package.json contents

```json
{
  "name": "okta-app",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@okta/okta-auth-js": "5.3.0",
    "@okta/okta-react": "^6.1.0",
    "@okta/okta-signin-widget": "^5.9.3",
    "@testing-library/jest-dom": "^5.14.1",
    "@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router-dom": "^5.2.0",
    "react-scripts": "4.0.3",
    "web-vitals": "^1.1.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

minimum viable reproduction of the issue. don't forget to update config.js with your urls and clientIds

https://github.com/msherman/okta-auth-example

**Comment by oleksandrpravosudko-okta** [comment]
Thanks for reporting this @msherman.

It looks like `userAgent` property is no longer present in the Auth SDK instance - downstream SDKs(react) will need to have their userAgent checks adjusted.

For now, please use `okta-auth-js@5.2.3`. Apologies for inconvenience.

Internal ref:  OKTA-419359

**Comment by shuowu** [comment]
Issue resolved within version 5.3.1.
